test_responses.py

```diff
@@ -399,7 +400,10 @@ def run():
         responses.add(responses.GET, url, body="test", status=418)
         resp = requests.get(url)
         assert resp.status_code == 418
-        assert resp.reason is None
```

Comment:
I'm not sure if the code is 'arbitrary' now, though. I suppose the purpose might be to use a code that isn't supported.

Reply:
Good point :+1: 
I changed the status code to `419`